packages/kg-default-nodes/lib/nodes/embed/types/twitter.js

```diff
@@ -92,6 +94,40 @@ export default function render(node, document, options) {
             }, '');
         }
 
+        // for compatibility with new api provider
+        if (tweetData && source === 'rettiwt') {
+            const wrapWithStyle = (text, style) => `<span style="${style}">${text}</span>`;
+        
+            let formattedContent = tweetContent.replace(/\n/g, '<br>');
+        
+            const tcoLinks = formattedContent.match(/https:\/\/t\.co\/[a-zA-Z0-9]+/g) || [];
+            const displayUrls = urls.map(urlObj => urlObj.display_url);
+            tcoLinks.forEach((tcoLink, index) => {
+                if (index < displayUrls.length) {
+                    formattedContent = formattedContent.replace(
+                        tcoLink,
+                        wrapWithStyle(displayUrls[index], 'color: #1DA1F2; word-break: break-all;')
+                    );
+                }
+            });
```

Comment:
With Rettiwt we don't have the start / end positioning of links like we have with the official API, in order to replace the shortened link with the actual link for better aesthetics.
However we do get an array with the urls from the tweet eg `['https://url1.com', 'https://url2.com']` and based on manual experimenting they appear in the order of which links appear in a post - however I don't think that's guaranteed based on the documentation.

Do you think it's worth the risk of a potential bad render should the ordering not render in the order as expected?